src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/OffenderProgramProfile.kt

```diff
@@ -39,7 +39,7 @@ data class OffenderProgramProfile(
   var program: ProgramService,
 
   @Column(name = "OFFENDER_START_DATE")
-  val startDate: LocalDate? = null,
```

Comment:
This column cannot be null even though table column is nullable. So point in wasting time catering for null values.

Reply:
and I guess you never read this column and need to read old bad NOMIS data?